src/PageContent/AssignVolunteers/AssignVolunteersController.js

```diff
@@ -0,0 +1,216 @@
+import React, { Component } from 'react';
+import './AssignVolunteers.css';
+import Edit from './Edit';
+import Confirmation from './Confirmation';
+import AssignVolunteersIndex from './AssignVolunteersIndex';
+
+class AssignVolunteersController extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            step: 0,
+            deliveries: [],
+            selectedDelivery: -1,
+            s1name: '',
+            s1phone: '',
+            s1email: '',
+            s2name: '',
+            s2phone: '',
+            s2email: ''
+        };
+    }
+
+    // TODO: Manually setting the values for now. Values would need to be queried from DB
+
+    componentWillMount() {
```

Comment:
ComponentDidMount does not work in this case because I am assuming the state is being updated and the rendering of the component depends on the updated state. There might be a way around this while working with backend code.